amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -1,5 +1,9 @@
 @import 'variables';
 
+$issue-banner-height: 40px;
+$shimmer-loader-height: 90px;
```

Comment:
nit nit: what do you think about increasing this to `120px` so that the shimmer loader lines will take up the same amount of space as the banner? I don't have some official methodology behind this, just a preference that sizing the shimmer items to the content improves that "feel" of content popping in nicely when ready.

Reply:
Yeah, I had some back and forth with Knowl about this. At the end we settled on not using this fixed size and use the fixed size of each line. I will update this code now.

Reply:
Fine with me, based on whatever Knowl recommended.

Reply:
We ended up having only two lines here.